client/settings/express-checkout/index.js

```diff
@@ -208,16 +247,43 @@ const ExpressCheckout = () => {
 						</div>
 					</li>
 					{ displayLinkPaymentMethod && (
-						<li className="express-checkout has-icon-border">
-							<div className="express-checkout__checkbox loadable-checkbox label-hidden">
-								<CheckboxControl
-									label={ __(
-										'Link by Stripe',
-										'woocommerce-payments'
-									) }
-									checked={ isStripeLinkEnabled }
-									onChange={ updateStripeLinkCheckout }
-								/>
+						<li className="express-checkout">
+							<div className="express-checkout__checkbox">
+								{ isPlatformCheckoutEnabled ? (
+									<Tooltip
+										content={ __(
+											'To enable Link by Stripe, you must first disable WooPay.',
+											'woocommerce-payments'
+										) }
+									>
+										<div className="loadable-checkbox__icon">
+											<Icon
+												icon={ warning }
+												fill={ '#ffc83f' }
+											/>
+											<div
+												className="loadable-checkbox__icon-warning"
+												data-testid="loadable-checkbox-icon-warning"
+											>
+												<VisuallyHidden>
+													{ __(
+														'Link by Stripe cannot be enabled at checkout. Click to expand.',
+														'woocommerce-payments'
+													) }
+												</VisuallyHidden>
```

Comment:
I added it so that screen readers can identify what the purpose of the tooltip.
The tooltip renders a `button` element around the SVG icon. The SVG icon is completely removed from the screen reader's text. So, without the `VisuallyHidden` element, a screen reader user would only "see" a `button` element with no text.
Adding a `VisuallyHidden` element adds some text for SR users, so that they can better understand the purpose of the `button` element.
An alternative could also have been to add an `aria-label` attribute on the `button` element.

I noticed that this article gives a decent overview of the problem: https://www.sarasoueidan.com/blog/accessible-icon-buttons/